All EIPIP Meetings/Meeting 060.md

```diff
@@ -0,0 +1,121 @@
similar to the owner of in erp 721 uh in eip721 owner of if token id doesn't exist then it reverts but in the this eip4907 the equivalent for effective user uh does not actually refer it just returns the zero address instead and uh and basically it'd be nice if they both reverted uh sure but so so if there's a final yeah so if it's final and like like having final eips that suck is pretty normal um and we have pretty strict policy that you know if your specification is final you've not changed the specification like my general way to phrase this is if someone implemented the eap before the change it must still be completely compliant after the change like you cannot have any possible implementation that becomes non-compliant due to this change and if understanding correctly this change would make something that was previously compliant now non-compliant so therefore they need a new eip yeah i just suggest sticking a new eip with that change in perhaps it could i'd say it'd probably be had to be merged in this review i'm not sure if it'd be allowed to be merged in this last call that might be a bit too ambitious but i wouldn't be entirely opposed to that i generally prefer people go through the process draft review last call final though i also don't care how fast i go through that process the main reason is is uh i just having to go through those steps there are some people that only pay attention on certain steps and so you skip a step you skip a set of people to pay attention like for example for ercs i will glance once at an rc when it's in draft and i will glance once when it goes into the last call and i ignore everything else and so if you skip one or more steps then i will not see it and so i like to at least go through the process but again it can be quick like you can just do submit it as a draft get it merged immediately submit to review as soon as that's merged merge it immediately send it submit it as last call merge it um and then wait for two weeks and submit to final like i don't expect anyone to take a certain amount of time besides the last call step i would agree that is generally a good practice here to start from draft and move it fast if that proposal is in good shape and people are accepting it the way it is probably i can add that as a comment so if the user who has uh suggested this change or has pointed it out maybe want to create a new proposal uh point number c 4c here is reference implementation aren't linted prior checking when marked as final i see this is um by a user tim top and i have seen some comment left by panda pape um anyone would like to summarize or is there any final decision on this issue so i think it's still very much debatable i would say that they should be all cc0 unless there is a good reason for them not to be for example the rfc's obviously we can't stick it cc0 um but i'd also suggest maybe having to have editor approval to have something that is non ccri i see point number five which is also on
```

Comment:
```suggestion
**Greg:**  well that's good so  **[we](https://www.youtube.com/watch?v=OlKWquEmJms&t=2803) want a button that says new eip and it opens up the topic with some minimal instructions um and fill in the blanks and um the author can take it from there** and this will encourage them to go read eip1 and use the template if they haven't already um i like that

 **Pooja**:  yeah i hope that that is going to be very helpful all right we have just 10 minutes left um i see a lot of issues there i mean like the topics to be covered let me try to bring some of them quickly 
 
 # 4.  From GitHub Issue
 ### a. Include a copy of RFC 822 and RFC 2119 in EIP-1's assets folder
 [number](https://www.youtube.com/watch?v=OlKWquEmJms&t=2803s) four from github issue um this is about um some conflict include a copy of rfc 822 and rfc2119 i have added the issue link here probably there are some discussion i guess initiated by pandapip1  
 
**Micah**:  i have no problem 

**Gavin**: sorry i didn't catch which one the issue was oh is this including the rfcs well i obviously don't have any issue with it 

**Micah**: yeah that one

 **Sam**:  so i'm in favor of adding two one one nine um i'm not sure if we're allowed to add the other one 
 
 **Micah**: what is the other one then why would we not me is that a different licensing yeah so 2119 explicitly says distribution of this memo is unlimited which according to their faq means that you can just redistribute it however you want um whereas the other one i think was 822 or something like that doesn't include that note so they don't give legal guidance on whether they're allowed to we're allowed to redistribute it or not 
 
 **Micah**: i interpret don't give leave bill guidance as do whatever you want but i don't know what the rest of you do no one tells me explicitly i can't do it i'm gonna do it 
 
 **Gavin**: also worst case they tell us take it down and we take it down 
 
 **Sam**: very true yeah 
 
 **Micah**: so i'm okay with approving that pr if no one else feels comfortable approving that pr um you know what so if someone created this pr i would have no problem approving it as long as no one rejected it if someone actually rejects it then i will respect that in further discussion uh do we put them in the assets directory for eip1 or do they go somewhere else 
 **Gavin**: Assets directory for EIP-1.
 
  **Sam** yeah perfect okay 
  
  **Micah** that's what i would do 
  
  ### b. [EIP-4907](https://eips.ethereum.org/EIPS/eip-4907) has inconsistencies with Requires (EIP-721), can it be updated?
 **Pooja** cool point number b here is for eip4907 that has inconsistencies which requires ip721 so here is the thing in the proposal 4907 um that has been moved to final it suggests that it requires eip721 as mandatory but the proposal has some inconsistencies with the required proposal 721. i'm dropping the link to the proposal here 
 
 **Micah**: does anyone know the technical history behind this 
 
 **Gavin**: so from what i gather uh the thing that's similar to the owner of in erp 721 uh in eip721 owner of if token id doesn't exist then it reverts but in the this eip4907 the equivalent for effective user uh does not actually refer it just returns the zero address instead and uh and basically it'd be nice if they both reverted 
 
 **Macah**: uh sure but so so if there's a final yeah so if it's final and like like having final eips that suck is pretty normal um and we have pretty strict policy that you know if your specification is final you've not changed the specification like my general way to phrase this is if someone implemented the eip before the change it must still be completely compliant after the change like you cannot have any possible implementation that becomes non-compliant due to this change and if understanding correctly this change would make something that was previously compliant now non-compliant so therefore they need a new eip 
 
 **Gavin** yeah i just suggest sticking a new eip with that change in perhaps it could i'd say it'd probably be had to be merged in this review i'm not sure if it'd be allowed to be merged in this last call that might be a bit too ambitious but i wouldn't be entirely opposed to that
 
**Micah**:  i generally prefer people go through the process draft review last call final though i also don't care how fast i go through that process the main reason is is uh i just having to go through those steps there are some people that only pay attention on certain steps and so you skip a step you skip a set of people to pay attention like for example for ercs i will glance once at an erc when it's in draft and i will glance once when it goes into the last call and i ignore everything else and so if you skip one or more steps then i will not see it and so i like to at least go through the process but again it can be quick like you can just do submit it as a draft get it merged immediately submit to review as soon as that's merged merge it immediately send it submit it as last call merge it um and then wait for two weeks and submit to final like i don't expect anyone to take a certain amount of time besides the last call step

 **Pooja**: i would agree that is generally a good practice here to start from draft and move it fast if that proposal is in good shape and people are accepting it the way it is probably **i can add that as a comment so if the user who has uh suggested this change or has pointed it out maybe want to create a new proposal** 
 
 ### c. Reference implementations aren't linted prior check-in or when marked as final
 **Pooja**: point number c 4c here is reference implementation aren't linted prior checking when marked as final i see this is um by a user tim  and i have seen some comment left by pandapip um anyone would like to summarize or is there any final decision on this issue 
 
 **Gavin**: so i think it's still very much debatable i would say that they should be all cc0 unless there is a good reason for them not to be for example the rfc's obviously we can't stick it cc0 um but i'd also suggest maybe having to have editor approval to have something that is non cc0.
```